### PR TITLE
Release wazuh 3.12.0_7.6.1

### LINKED ISSUE #315: Wazuh Release 3.12.0_7.6.1
### Wazuh Release
**Wazuh version**: 3.12.0
**Elastic version**: 7.6.1

### Tasks

- [x] Adapt to new versions (3.12.0 - 7.6.1)

- [x] Check templates and update them if needed

- [x] Tests

- [x] Update changelog

- [x] Tag: **v3.12.0_7.6.1**

- [x] Update documentation references

- [ ] Draft release

### PR DESCRIPTION
**Pull request by Zenidd**
This PR closes #315

`Releasing 3.12.0_7.6.1`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+
+## Wazuh Docker v3.11.4_7.6.1
```
Comment (manuasir) [inline CHANGELOG.md]: This has to point `v3.12.0`

File: docker-compose.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Wazuh App Copyright (C) 2019 Wazuh Inc. (License GPLv2)
+# Wazuh App Copyright (C) 2020 Wazuh Inc. (License GPLv2)
 version: '2'
 
 services:
   wazuh:
-    image: wazuh/wazuh:3.11.4_7.6.1
+    image: wazuh/wazuh:3.12.4_7.6.1
```
Comment (manuasir) [inline docker-compose.yml]:
```suggestion
    image: wazuh/wazuh:3.12.0_7.6.1
```

File: VERSION [hunk 3]
```diff
@@ -1,2 +1,2 @@
-WAZUH-DOCKER_VERSION="3.11.4_7.6.1"
-REVISION="31140"
+WAZUH-DOCKER_VERSION="3.12.0_7.6.1"
+REVISION="31204"
```
Comment (xr09) [inline VERSION]: This should be `31200`

File: CHANGELOG.md (hunk 1 above)
Comment (Zenidd) [reply CHANGELOG.md]: Fixed for all tags in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetins !

File: docker-compose.yml (hunk 2 above)
Comment (Zenidd) [reply docker-compose.yml]: Fixed for all tags in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetings !

File: VERSION (hunk 3 above)
Comment (Zenidd) [reply VERSION]: Fixed in https://github.com/wazuh/wazuh-docker/pull/316/commits/cb60fc8b771a04b2c3beb652c305ffd5b316687e

Greetings !

**Comment by ilhaan** [comment]
Just tested Kibana from this PR and it seems like `hosts:` needs to be added to the Wazuh config file as shown below: 
```
if [[ "x$CONFIG_CODE" != "x200" && $_config_exists -ne 0 ]]; then
cat << EOF >> $kibana_config_file 
hosts:
  - 1513629884013:
      url: $wazuh_url
      port: $wazuh_port
      user: $api_user
      password: $api_password
EOF
else
  echo "Wazuh APP already configured"
fi
```
I made the above shown change to `config/wazuh_app_config.sh` to get Wazuh working correctly. I can't find this in the [documentation](https://documentation.wazuh.com/3.12/user-manual/kibana-app/reference/config-file.html) but navigating to Wazuh in the Kibana UI showed an example that used `hosts` as shown above.

**Comment by xr09** [comment]
Awesome! Thank you for the heads up @ilhaan, I'm on it!

**Pull request merged**